_utilities/layout-grid.md

```diff
@@ -504,116 +357,150 @@ Add `grid-gap` to a grid row to add a gap (or gutter) between each column in the
+  {% include utilities/utility-mixin-intro.html %}
 
-{% highlight scss %}
-.example-container {
-  @include grid-container;
-}
-
-.example-row {
-  @include grid-row;
-
-  // Add column gaps
-  &.content-row {
-    @include grid-gap;
-  }
-}
-
-.example-content-main {
-  @include u-width(full);
-
-  @include at-media(tablet) {
-    @include grid-col(6);
-  }
-
-  @include at-media(desktop) {
-    @include grid-col(8);
-  }
-}
+  <table class="usa-table--borderless site-table-responsive site-table-simple">
+    <thead>
+      <tr>
+        <th scope="col" class="tablet:maxw-card-lg display-inline-flex flex-2">Utility</th>
+        <th scope="col" class="display-inline-flex flex-2">Mixin</th>
+        <th scope="col" class="display-inline-flex flex-1">Accepted values</th>
+        <th scope="col" class="display-inline-flex flex-2">Example</th>
+      </tr>
+    </thead>
+    <tbody class="font-mono-2xs">
+      <tr>
+        <td data-title="Utility" class="tablet:maxw-card-lg flex-2">
+          <span>
+            .grid-container <br />
+            .grid-container-<code>container size</code>
+          </span>
+        </td>
+        <td data-title="Mixin" class="display-inline-flex flex-2">
+          <span>
+            grid-container<code>(container size)</code>
+          </span>
+        </td>
+        <td data-title="Accepted values" class="display-inline-flex flex-1">
+          <span>
+            mobile, mobile-lg, tablet, tablet-lg, desktop, desktop-lg, or widescreen
+          </span>
+        </td>
+        <td data-title="Example" class="display-inline-flex flex-2">
+          <span>
+            @include grid-container <br/>
+            @include grid-container("tablet")
+          </span>
+        </td>
+      </tr>
+      <tr>
+        <th scope="row" data-title="Utility" class="tablet:maxw-card-lg display-inline-flex flex-2">
+          <span class="text-normal">
+            .grid-row
+          </span>
+        </th>
+        <td data-title="Mixin" class="display-inline-flex flex-2">
+          <span>
+            grid-row
+          </span>
+        </td>
+        <td data-title="Accepted values" class="display-inline-flex flex-1">
+          <span>
+            -
+          </span>
+        </td>
+        <td data-title="Example" class="display-inline-flex flex-2">
+          <span>
+            @include grid-row
+          </span>
+        </td>
+      </tr>
+      <tr>
+        <th scope="row" data-title="Utility" class="tablet:maxw-card-lg display-inline-flex flex-2">
+          <span class="text-normal">
+            .grid-col <br />
+            .grid-col-<code>column count</code>
+          </span>
+        </th>
+        <td data-title="Mixin" class="display-inline-flex flex-2">
+          <span>
+            grid-col<code>(column count)</code>
+          </span>
+        </td>
+        <td data-title="Accepted values" class="display-inline-flex flex-1">
+          <span>
+            sm, md, lg, 2px, 05, 1, 3, 4, 6
```

Comment:
**issue**: The accepted values listed under grid col look incorrect.

![image](https://github.com/user-attachments/assets/a739bcfc-d1c8-475e-a3a1-d9688f8b5d62)

**Expected values**
```
auto, fill, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12
```

Reply:
Thanks, looks like some of these values got moved around during edits.

Resolved in https://designsystem.digital.gov/design-tokens/order/